### DISCUSSION TITLE
Open an unknown file extension with a specific syntax highlighting

### DISCUSSION THREAD

**Discussion by cheridanis** [phcode-dev/phoenix#1786] [Q&A] [answered]
There are file extensions which are specific to some web development platforms (such as ".f7" from framework7). These extensions are unknown to brackets or phoenix. It would be very nice if I can choose which syntax highlighting to use when opening those files .

**Comment by abose** [comment, marked as answer]
It's curretnly possible to do that. Click on the language dropdown in the status bar.
![image](https://github.com/user-attachments/assets/c708279f-2ede-42f1-9867-35f20652ca02)

**Reply by cheridanis** [reply]
Thank you, I didn't notice how easy it is to do it😁
